### ISSUE TITLE
Scrcpy error ( Remi note 7)

### ISSUE THREAD

**Issue by nhathuypopo** [Genymobile/scrcpy#1567]
![image](https://user-images.githubusercontent.com/67586698/86528870-5e254100-bed6-11ea-8e1f-1d28b97efc32.png)
INFO: scrcpy 1.14 <https://github.com/Genymobile/scrcpy>
C:\Users\Huy\Downloads\scrcpy-win64-v1.14\scrcpy-server: 1 file pushed, 0 skipped. 46.7 MB/s (33142 bytes in 0.001s)
[server] INFO: Device: Xiaomi Redmi Note 7 (Android 9)
INFO: Created renderer: direct3d
INFO: Renderer: direct3d
INFO: Initial texture: 1080x2336
ERROR: SDL failed to get a vertex buffer for this Direct3D 9 rendering batch!
ERROR: Dropping back to a slower method.
ERROR: This might be a brief hiccup, but if performance is bad, this is probably why.
ERROR: This error will not be logged again for this renderer.
 I have these error how can i fix it?

**Comment by rom1v** [comment]
#1441 

TL;DR: just ignore it

**Comment by nhathuypopo** [comment]
but i cant control my phone on my computer

**Comment by rom1v** [comment]
https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#mouse-and-keyboard-do-not-work
